app/src/main/res/layout/activity_fragment.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".Fragment.FragmentActivity">
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/btn_fragment_simple"
+        android:background="#ADD8E6"
+        android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:layout_marginLeft="10dp"
+        android:layout_marginRight="10dp"
+        android:layout_marginTop="60dp"
+        app:layout_constraintTop_toTopOf="parent"
+        android:text="@string/btn_chap4_title"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+         />
+    <androidx.appcompat.widget.AppCompatButton
+        android:id="@+id/btn_fragment_fragmentcontainer"
```

Comment:
use only one type of id naming in through out the project, use either camel case or snake case, not both